.github/workflows/pr-comment-bot.yml

```diff
@@ -0,0 +1,18 @@
+name: PR comment bot
+on:
+  issue_comment:
+    types: [created]
+jobs:
+  pr-comment:
```

Comment:
https://github.com/sendbird/release-automation-action#commands 
good to have this explicit ~ as comment

Reply:
cc @bang9

Reply:
Could you please explain in a bit more detail?